pepper-apis/src/main/resources/db-changes/schema/DDP-5910-pdf-picklist-fields-mapping.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet id="20181003-pdf-picklist-fields-mapping" author="ypavlov">
+        <createTable tableName="pdf_picklist_field_mapping">
+            <column name="pdf_substitution_id" type="bigint">
+                <constraints nullable="false"
```

Comment:
Should we set this as the primary key for this table as well?

Reply:
It's not a primary key and not unique

Reply:
Oh right, a substitution can have many mappings.